tests/app/services/test_query_builder.py

```diff
@@ -138,36 +139,36 @@ def test_should_have_filtered_term_lot_clause():
         "saas")
 
 
-def test_should_have_filtered_term_for_lot_and_service_types_clause():
-    query = construct_query(
-        build_query_params(filters={'lot': "SaaS", 'serviceTypes': ["serviceTypes"]}))
+def test_should_have_filtered_term_for_lot_and_service_types_clause(services_mapping):
+    query = construct_query(services_mapping,
+                            build_query_params(filters={'lot': "SaaS", 'serviceTypes': ["serviceTypes"]}))
     terms = query["query"]["bool"]["filter"]["bool"]["must"]
     assert_in({"term": {'filter_serviceTypes': 'servicetypes'}}, terms)
     assert_in({"term": {'filter_lot': 'saas'}}, terms)
 
 
-def test_should_not_filter_on_unknown_keys():
+def test_should_not_filter_on_unknown_keys(services_mapping):
     params = build_query_params(filters={'lot': "SaaS", 'serviceTypes': ["serviceTypes"]})
     params.add("this", "that")
-    query = construct_query(params)
+    query = construct_query(services_mapping, params)
     terms = query["query"]["bool"]["filter"]["bool"]["must"]
     assert_in({"term": {'filter_serviceTypes': 'servicetypes'}}, terms)
     assert_in({"term": {'filter_lot': 'saas'}}, terms)
     assert_not_in({"term": {'unknown': 'something to ignore'}}, terms)
 
 
-def test_should_have_filtered_term_for_multiple_service_types_clauses():
-    query = construct_query(
-        build_query_params(
-            filters={'serviceTypes': ["serviceTypes1", "serviceTypes2", "serviceTypes3"]}))
+def test_should_have_filtered_term_for_multiple_service_types_clauses(services_mapping):
+    query = construct_query(services_mapping,
+                            build_query_params(filters={
+                                'serviceTypes': ["serviceTypes1", "serviceTypes2", "serviceTypes3"]}))
```

Comment:
There are some odd combinations of single and double quotes in these tests, but I get that this is all legacy :)

Reply:
I'm not on track to make things better cos I have no idea which one is preferred, sorry! What's your pref?!